actors/miner/src/lib.rs

```diff
@@ -1981,11 +1985,14 @@ impl Actor {
             activate_sectors_deals(rt, &data_activations, compute_commd)?;
         let successful_activations = batch_return.successes(&precommited_sectors);
 
+        // TODO: total_power is not requested here, so how can we get ramp_start_epoch & ramp_duration_epochs ?
```

Comment:
@lemmih - flag here, do we need to make a call to get_total_power(), or is this function a special startup state where the inputs are noops?

Reply:
You ran into some technical debt here. This whole method is legacy and never invoked on chain. It should be removed. It's used only for creating genesis state (e.g. for testnets). Just set both values to zero.

cc @ZenGround0